pkg/sql/ttl/ttljob/ttljob_test.go

```diff
@@ -317,7 +317,11 @@ func TestRowLevelTTLJobRandomEntries(t *testing.T) {
 			})
 			defer cleanupFunc()
 
+			rangeBatchSize := 1 + rng.Intn(3)
```

Comment:
if the default is 100, should the test be closer to that?

Reply:
we used a smaller number here here because we have less ranges (maximum of ~8).